static/js/components/OriginSelect.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SelectWithChips from "./SelectWithChips";
+
+const OriginSelect = ({ onOriginSelectChange, initValue, parent }) => {
+  const origins = ["Clear selections", "Muphoten", "STDpipe"];
```

Comment:
Are these default values?

Reply:
@thomasculino were these placeholder values? and fetching all the origins is a WIP?

Reply:
Yes, these are default values. We discussed that it could be a good idea to have default values for origin because fetching all the origins would mean fetching all photometry and then we will retrieve all unique origins. We would also add a field for users to add a custom origin if it is not in the list. This way, we can avoid fetching all photometry. Maybe we can find another way to do this but this was my reasoning.

Reply:
@guynir42 thoughts?

Reply:
@jadalilleboe @thomasculino the `origin` column is indexed, so it seems to me it should not be terribly expensive to query the `distinct()` values even on all photometry points. You may want to be selective on the columns you are getting but it may not be so bad. 

Since this is a dropdown menu we don't expect people to use all the time, it would not be a big issue if it takes a couple of seconds to populate it. I suggest first of all trying it with a reasonable query and testing it out on production. If it turns out to be annoyingly slow, we can figure out something smarter to do. 

The smarter thing would probably be to cache all incoming photometry origins, into a set or something. But I have a feeling that this is exactly what the index is already doing, so probably this query is already going to be optimized.

Reply:
@guynir42 should we get this merged and fetching the origins can be a separate PR? (maybe @thomasculino will have time to work on that over the summer, I'll be starting my fulltime job soon)

Reply:
Yeah a separate PR is fine by me.